node.go

```diff
@@ -251,28 +251,84 @@ func (n *Node) RequestNodeInformationFrame() error {
 	return err
 }
 
+// To optimize the interview process we'll be interview command classes
+// in the following order:
+// 1) SecurityCommandClass
+// 2) VersionCommandClass
+// 3) Remaining insecure command classes
+// 4) Secure command classes
+func commandClassesInOrderToInterview(set cc.CommandClassSet) []cc.CommandClassID {
+	insecure := set.ListBySecureStatus(false)
+
+	securityccidx := -1
+	versionccidx := -1
+	remaininginsecure := []cc.CommandClassID{}
+	for idx, ccid := range insecure {
+		if isVersionCommandClassID(ccid) {
+			versionccidx = idx
+			continue
+		}
+		if isSecurityCommandClassID(ccid) {
+			securityccidx = idx
+			continue
+		}
+		remaininginsecure = append(remaininginsecure, ccid)
+	}
+
+	if securityccidx > -1 && versionccidx > -1 {
+		insecure = append([]cc.CommandClassID{
```

Comment:
Is it worth handling the case where there is a version command class but no security cc?

Reply:
I'd like to avoid doing that. I was definitely not able to pair if I interviewed the VersionCommandClass first, so I don't know if it's ever a good idea to hard code it to put that interview first.

Reply:
Agreed we probably don't want it first for these devices, given that they require the security CC. However, it seems that this would break any device that _intentionally_ never reported the Security command class.

Reply:
It wouldn't break it, it would just not attempt to reorder the command classes.

Reply:
There's a unit test for this case

Reply:
Gotcha, makes sense :+1: